elasticapm/conf/__init__.py

```diff
@@ -563,6 +563,7 @@ class Config(_ConfigBase):
     prometheus_metrics = _BoolConfigValue("PROMETHEUS_METRICS", default=False)
     prometheus_metrics_prefix = _ConfigValue("PROMETHEUS_METRICS_PREFIX", default="prometheus.metrics.")
     disable_metrics = _ListConfigValue("DISABLE_METRICS", type=starmatch_to_regex, default=[])
+    disable_metrics_thread = _BoolConfigValue("DISABLE_METRICS_THREAD", default=False)
```

Comment:
In other parts of the code, we take a metrics interval of 0 as an indication that the metrics thread should not be started, e.g.

https://github.com/elastic/apm-agent-python/blob/b33f4494d5120612c43b8ee76aa26833543ad65a/elasticapm/metrics/base_metrics.py#L94

We should document that fact though, currently it isn't: https://github.com/elastic/apm-agent-python/blob/fdc0bf03a5bd46449c5ab579accb6514b2c8a2b6/docs/configuration.asciidoc#metrics_interval

Reply:
Good callout, thanks.